config/god/sidekiq_workers.rb

```diff
@@ -17,6 +17,10 @@ def start_cmd(queues, concurrency)
   end
 
   def pid_file
-    File.join(RAILS_ROOT, "tmp/pids/sidekiq-#{count}.pid")
+    if ! defined?(AppConfig)
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions.<br>Style/SpaceAfterNot: Do not leave space between `!` and its argument.